### PR TITLE
Feature/94/header

### PR DESCRIPTION
**Pull request by sudo-eddy**
### Describe the problem being solved:
Navbar and navbar responsiveness
### Impacted areas in the application: 
Navbar and CSS classes 

List general components of the application that this PR will affect: 
* Overall site responsiveness

PR checklist
- [x] I included  a screenshot for FE changes
- [x] I have linked the PR to a Zenhub ticket
- [x] I have checked for merge conflicts

Screenshot from 2019-05-19 10-36-43

### CODE REVIEW

**Comment by sudo-eddy** [comment]
## Things to work on
  

- I had a discussion with @Zgarcia72292 about site responsiveness in smaller screens. We think it's a good idea to hide the logo and show the top navbar when displaying in mobile. Maybe we can revisit this later. We also talked about moving the social media icons to the bottom of the page instead of showing them in the collapsed navbar when displayed in mobile devices.
Found some good insight [here](https://hackernoon.com/conditional-rendering-on-react-57a864c2d04d).

- Collapsed navbas need a bit more styling and a higher z-index.

- Style silver circle under logo

- Logo touches the navbar when screen size @1024px

- Carousel needs some work as well, wondering if we should wait to get better images from the client (without the logo/text)

## I wanted to PR my changes because I think it's a good idea to get more team members involved in this ticket and I would love some help with these small changes.

File: frontend/src/actions/events.js [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
-      .then(res => res.json())
-      .then(events => {
-        return dispatch({
-          type: "FETCH_EVENTS",
-          events
-        });
-      });
-  };
-};
+// export const fetchEvents = () => {
```
Comment (mkirlin) [inline frontend/src/actions/events.js]: None of this should be commented out; please uncomment it.

File: frontend/src/components/Header/Header.css [hunk 2]
```diff
@@ -1,38 +1,121 @@
...
-  align-items: center;
-  margin: 0;
+	list-style: none;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	margin: 0;
+}
+
+.nav-linkb {
```
Comment (mkirlin) [inline frontend/src/components/Header/Header.css]: This needs a more descriptive name.

File: frontend/src/components/Header/Header.css [hunk 3]
```diff
@@ -1,38 +1,121 @@
...
+	padding: 20px;
+	color: black;
+	height: 100%;
 }
 
 .nav-link {
-  padding: 20px;
-  color: hsl(101,43%,42%);
-  height: 100%;
+	padding: 20px;
```
Comment (mkirlin) [inline frontend/src/components/Header/Header.css]: I mentioned this in @Aleksandandar-Nedelkovski's PR #107 ; this is going to cause issues. Remove this styling from the `nav-links` class and apply a new class to the elements you want to style.

File: frontend/src/components/Header/Header.css [hunk 4]
```diff
@@ -1,38 +1,121 @@
...
+.nav-link-top {
+	padding: 20px;
+	color: #000000;
+	height: 100%;
 }
 
 .nav-link:hover {
-  background: hsl(95,60%,56%);
-  color: white;
+	color: white;
```
Comment (mkirlin) [inline frontend/src/components/Header/Header.css]: Same thing here; any Bootstrap classes should not be modified.

File: frontend/src/components/Header/NavTop.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
```
Comment (mkirlin) [inline frontend/src/components/Header/NavTop.jsx]: It looks like you've removed all the admin logic; we're going to need that, so please add it back in here.

File: frontend/src/components/Midsection/LowerMid.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import "./mid.css";
+
+
+class LowerrMid extends Component {
```
Comment (mkirlin) [inline frontend/src/components/Midsection/LowerMid.jsx]: This is misspelled.

File: frontend/src/components/Midsection/LowerMid.jsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    render() {
+        return (
+            <div className="lowermid">
+                <div className="container">
```
Comment (mkirlin) [inline frontend/src/components/Midsection/LowerMid.jsx]: This should not be a container.

File: frontend/src/components/Midsection/LowerMid.jsx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+                        <button className='text-center ym-button' id="enroll">All Events</button>
+                    </div>
+                </div>
+            </div>
+        );
+
+    }
+}
+
+export default LowerrMid;
```
Comment (mkirlin) [inline frontend/src/components/Midsection/LowerMid.jsx]: Also misspelled here.

File: frontend/src/components/Midsection/UpperMid.jsx [hunk 9]
```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import "./mid.css";
+
+
+class UpperMid extends Component {
+    render() {
+        return (
+            <div>
+                <div className="container">
```
Comment (mkirlin) [inline frontend/src/components/Midsection/UpperMid.jsx]: This should also not be a container.

File: frontend/src/components/footer.css [hunk 10]
```diff
@@ -2,8 +2,26 @@
...
+
+.menu-text {
+  color: white;
+}
+
+.foot_text {
+  color: white;
+  text-decoration: none;
+}
+li {
```
Comment (mkirlin) [inline frontend/src/components/footer.css]: Please don't style default components. Apply a class instead.

File: frontend/src/components/footer.css [hunk 11]
```diff
@@ -2,8 +2,26 @@
...
+  color: white !important;
+  text-decoration: none !important;
+}
+
+nav {
```
Comment (mkirlin) [inline frontend/src/components/footer.css]: Ibid.

File: frontend/src/components/footer.jsx [hunk 12]
```diff
@@ -11,22 +11,29 @@ export default class footer extends Component {
             <footer className='footer py-3'>
                 <div className='container'>
                     <nav>
-                        <ul className='row'>
+                        <ul className='row menu-text'>
                             <li className='col footer_link'>
-                                <Link to="/about">About Us</Link>
+                                <Link to="/about" style={{ color: 'white', fontSize: "12px" }} className="foot-text">About</Link>
```
Comment (mkirlin) [inline frontend/src/components/footer.jsx]: Also mentioned in #107, all of the in-line styling needs to go into a CSS file.

File: frontend/src/components/home.jsx [hunk 13]
```diff
@@ -1,14 +1,16 @@
...
 import "./home.css";
 
 class Home extends Component {
   render() {
     return (
-      <div className="Home container">
-        <h1>YMIM</h1>
+      <div className="Home">
         <CarouselComponent />
+        <UpperMid />
```
Comment (mkirlin) [inline frontend/src/components/home.jsx]: All three of these should be in rows.

File: frontend/src/router.js [hunk 14]
```diff
@@ -25,10 +25,10 @@ let store = createStore(fetchData, applyMiddleware(thunk));
 const router = (
   <Provider store={store}>
     <Router>
-      <div className="site container-fluid">
+      <div className="site ">
         <div className="site-content">
           <Header />
-          <div className="container">
+          <div>
```
Comment (mkirlin) [inline frontend/src/router.js]: This should be wrapped in a container.

**Pull request closed without merging**